src/app/test/business/dad_jokes_service_test.dart

```diff
@@ -38,7 +39,11 @@ void main() {
     );
 
     favoriteDadJokesRepository = FavoriteDadJokesRepository();
-    SUT = DadJokesService(dadJokesRepository, favoriteDadJokesRepository);
+    SUT = DadJokesService(
+      dadJokesRepository,
+      favoriteDadJokesRepository,
+      Logger(),
```

Comment:
Should we use the same instance that we use normally?

Reply:
To be honest, I wanted to mock it just like we're doing it on .NET, but I find it easier to simply use the default Logger. To answer your question, I would avoid using `LoggerManager` to create logs here especially since we only want to test the dad jokes services